__tests__/jesttest.tsx

```diff
@@ -92,6 +94,55 @@ describe("LibraryContainer", function () {
     expect(libraryItem.prop('data').childItems[0].childItems[0].text).to.equal("Child1");
     expect(libraryItem.prop('data').childItems[0].childItems[1].text).to.equal("Child2");
   });
+   describe("Test event on libraryitem", function () {
+        it("Must recognise mouse event and change state", function () {
```

Comment:
I think it would be better if the description of the test case can be more detailed, like `should recognise mouse click event and change the expand state`. Since there are several mouse events for one library item, this way there would be no confusion with other testcases.